README.md

```diff
@@ -63,13 +71,13 @@ curl -L https://github.com/openpubkey/opkssh/releases/latest/download/opkssh-lin
 
 ### SSHing with opkssh
 
-After downloading opkssh, on OSX or Linux run:
+After downloading opkssh run:
 
 ```cmd
 opkssh login
 ```
 
-on Windows run:
+on manual Windows install (using curl) run:
```

Comment:
I just tested this and `opkssh login` works for both the curl install and wget install! So we get rid of this additional instruction